app/presenters/two_factor_auth_code/sms_opt_in_presenter.rb

```diff
@@ -0,0 +1,9 @@
+module TwoFactorAuthCode
+  class SmsOptInPresenter < GenericDeliveryPresenter
+    def initialize; end
```

Comment:
(probably for another PR, just noting while I'm here)

this jumped out at me as unnecessary so I checked the superclass... that `instance_variable_set`... loop dynamically setting @-vars seems very unintuitive to me, I think that we might want to try removing that behavior? Two ideas:

1. Since clearly each subclass needs to be instantiated with very different data, remove `options` from the base class and let each sublcass have its own constructor
2. Make the DeliveryPresenter behavior into a mixin, let each subclass instantiate as needed?